content/updates/2020-08/3-API-option-labels.md

```diff
@@ -0,0 +1,202 @@
+---
+pim_announcement_img: ./img/option-labels.png
+pim_announcement_alt_img: "Option labels inside product endpoint"
+pim_announcement_audience:
+- CE
+- EE
+---
+
+# Get options labels directly inside the products endpoint
```

Comment:
```suggestion
# Get options labels directly inside the products endpoints
```
"products endpoints" instead of "products endpoint" ?

Reply:
But only the Get product/products endpoints are concerned by this feature right?

Reply:
I feel like talking about endpoints here won't necessarily be understandable for Julia. I would suggest something like:
```suggestion
# Options labels in product export via API
```